packages/ecs/src/ComponentFunctions.ts

```diff
@@ -561,61 +563,78 @@ export const setComponent = <C extends Component>(
   component.onSet(entity, component.stateMap[entity]!, args)
 
   const propagate = () => {
```

Comment:
this function isn't actually getting called so no layer logic is running in setComponent right now

Reply:
Yeah I disabled it for now, just trying to get loading into both layers working properly